inc/routes.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * REST endpoints for callback functions
+ *
+ * @package Theme_Sniffer
+ */
+
+add_action( 'rest_api_init', 'theme_sniffer_endpoint_init' );
+
+/**
+ * Register endpoints function
+ *
+ * @since 0.1.0
+ */
+function theme_sniffer_endpoint_init() {
+	register_rest_route( 'wp/v2/theme-sniffer/', '/sniff-run', array(
+		'methods'  => 'POST',
+		'callback' => 'theme_sniffer_run_sniffer',
+	) );
+
+	register_rest_route( 'wp/v2/theme-sniffer/', '/individual-sniff', array(
```

Comment:
@dingo-d As I was reading up on `register_rest_route()` I realized we should use `theme-sniffer/v1` as the namespace. Could you make a PR to change this?
https://developer.wordpress.org/rest-api/extending-the-rest-api/routes-and-endpoints/#namespaces
https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/#bare-basics

Reply:
Sure thing, PR coming right up.